mathesar_ui/src/components/abstract-type-control/DbTypeIndicator.svelte

```diff
@@ -1,17 +1,12 @@
 <script lang="ts">
   import { _ } from 'svelte-i18n';
   import type { DbType } from '@mathesar/AppTypes';
-  import { RichText } from '@mathesar/components/rich-text';
 
   export let type: DbType | undefined;
 </script>
 
 <div>
-  <RichText text={$_('database_type_displayer')} let:slotName>
-    {#if slotName === 'format'}
-      <span>{type ?? 'UNKNOWN'}</span>
-    {/if}
-  </RichText>
+  {$_('database_type')}: <span>{type ?? 'UNKNOWN'}</span>
```

Comment:
This seems like it ought to be one string like `Database type: {databaseType}`

Reply:
@seancolsen I separated them since this content appears in contexts similar to a form. Eg., `Name: <TextInput value={name} />`. I don't feel strongly about either option and I'm fine with both, we could make it into a single string if you feel strongly about it.

Reply:
I see what you mean. I certainly don't feel strongly about this, I just thought that was best practice. Maybe I'm wrong though. I see what you mean about it looking like a form. I guess it's fine since it's not like this is a complete sentence. I'll defer to you. It would be great to eventually have a guide for our devs that clarifies our best practice around some of the edge cases like this.

Reply:
> It would be great to eventually have a guide for our devs that clarifies our best practice around some of the edge cases like this.

Agreed. I intend to work on the guide once the remaining i18n work is complete.